{{cookiecutter.project_slug}}/app.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "{{ cookiecutter.project_name }}",
+    "env": {
+        "ALLOWED_HOSTS": {
+            "value": ".herokuapp.com"
+        },
+        "DEBUG": {
+            "value": "True"
+        },
+        "DISABLE_COLLECTSTATIC": {
+            "value": 1
+        },
+        "ENVIRONMENT": {
+            "value": "development"
+        },
+        "PROJECT_PATH": {
+            "value": "modat_api"
```

Comment:
@oudeismetis ruh roh. This should be generic

Reply:
ugh